sw-precache-config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  staticFileGlobs: [
+    'out/**/*.html',
+    'out/static/**/*',
+    'out/_next/**/*'
+  ],
+  stripPrefix: 'out/',
+  replacePrefix: '/2.0.0/',
+  verbose: true,
+  runtimeCaching: [{
+    urlPattern: /(.*)/,
+    handler: 'fastest'
```

Comment:
Use the `'fastest'` profile only for files inside `_next` and `static` folder. In other pages, use `networkFirst` handler. Otherwise, user may get stale pages :(